File: ch16/phpunit/dev/work/src/Tax.php
<?php

namespace Root\Root;

final class Tax
{
    public function calc_tax(int $price, bool $is_exempted, bool $is_reduced, bool $is_taken_out): int
    {
        if ($is_exempted)
        {
            return $price;
        }
        else if ($is_reduced && $is_taken_out)
        {
            return (int) ($price * 1.08);
        }
        else
        {
            return (int) ($price * 1.1);
        }
    }
}

File: ch16/junit/dev/work/app/src/main/java/root/Tax.java
package root;

public class Tax {
    public int calcTax(int price, boolean isExempted, boolean isReduced, boolean isTakenOut) {
        if (isExempted) {
            return price;
        } else if (isReduced && isTakenOut) {
            return (int) (price * 1.08);
        } else {
            return (int) (price * 1.1);
        }
    }
}
